discord/member.py

```diff
@@ -402,6 +402,22 @@ def permissions_in(self, channel):
         """
         return channel.permissions_for(self)
 
+    def has_role(self, role_id):
+        """Check if a user has a role with the given ID.
```

Comment:
Seems like we all missed this. It's a member.
```suggestion
        """Check if a member has a role with the given ID.
```